Makefile

```diff
@@ -7,7 +7,8 @@ help:
 	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-20s\033[0m %s\n", $$1, $$2}'
 
 ENVTEST_K8S_VERSION ?= 1.22.1
-ARCHITECTURE = amd64
+ARCHITECTURE ?= amd64
+PLATFORMS ?= linux/amd64,linux/s390x
```

Comment:
I don't think we want to build on `linux/s390x` every time. I would change this to default to `linux/amd64`, or even to the current architecture from the `uname` output.